awx/plugins/inventory/foreman.py

```diff
@@ -229,10 +236,6 @@ def _resolve_params(self, host_params):
 
         return params
 
-    def _get_facts_by_id(self, hid):
-        url = "%s/api/v2/hosts/%s/facts" % (self.foreman_url, hid)
-        return self._get_json(url)
```

Comment:
This method is still called by `_get_facts`, and the class inherits from `object`, so it is not available via some other parent class.

Reply:
it was in the file twice, it should still be in the resulting file

Reply:
oh I see that now

https://github.com/ansible/awx/blob/f1b1224a27faac66a2b9ea44cee5b5e13e894496/awx/plugins/inventory/foreman.py#L219